lib/raxx/kit.ex

```diff
@@ -42,14 +42,33 @@ defmodule Raxx.Kit do
       end
     end)
 
+    run_instructions =
+      case {config.docker, !!config.ecto} do
+        {true, _} ->
+          "    docker-compose up"
+
+        {false, false} ->
+          "    iex -S mix"
+
+        {false, true} ->
+          # the backslash at the end is not a typo, it's removing
+          # the newline from the end of the heredoc
+          """
+              nano config/config.exs # configure the #{config.module}.Repo database
```

Comment:
nano?

Reply:
I figured it was the common denominator of cli text editors. I'm not sure what's the best way of saying "edit this here file" in this format.

Reply:
fair enough. It will do for now can come up with something better perhaps later on